sklearn/kernel_approximation.py

```diff
@@ -262,12 +262,12 @@ class RBFSampler(_ClassNamePrefixFeaturesOutMixin, TransformerMixin, BaseEstimat
 
     Attributes
     ----------
-    random_offset_ : ndarray of shape (n_components,), dtype=float64
+    random_offset_ : ndarray of shape (n_components,), dtype=float64 or float32
```

Comment:
What is the best way to express "mostly float64, but float32 if X is float32"?

Reply:
I think it looks good like this. `numpydoc` validation seems to be green.